perfplot/main.py

```diff
@@ -72,6 +72,7 @@ def show(self):
     def save(self, filename, transparent=True, bbox_inches="tight"):
         self.plot()
         plt.savefig(filename, transparent=transparent, bbox_inches=bbox_inches)
+        plt.clf()
```

Comment:
Don't you want `plt.close()`?

Reply:
As you wish, but is there any difference in this case? The figure is not shown, just saved to file, and my intuition was that after cleaned it may be reused. I found [comment](https://stackoverflow.com/a/46957388) that states something opposite - suggests that plt.clf() may be less performant but I could neither confirm that in documentation or reproduce such warning.